net-mail/mu/mu-1.3.6.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools elisp-common
+
+DESCRIPTION="Set of tools to deal with Maildirs, in particular, searching and indexing"
+HOMEPAGE="http://www.djcbsoftware.nl/code/mu/ https://github.com/djcb/mu"
+SRC_URI="https://github.com/djcb/mu/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="emacs guile mug"
+
+# net-mail/mailutils also installes /usr/bin/mu.  Block it until somebody
+# really wants both installed at the same time.
+DEPEND="
+	dev-libs/gmime:3.0
+	>=dev-libs/xapian-1.4
+	dev-libs/glib:2
+	emacs? ( >=app-editors/emacs-24.4:* )
+	mug? ( >=net-libs/webkit-gtk-2.0 >=x11-libs/gtk+-3.0 )
+	guile? ( >=dev-scheme/guile-2.0 )
+	!net-mail/mailutils"
+RDEPEND="${DEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+SITEFILE="70mu-gentoo.el"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf $(use_enable emacs mu4e) \
+		$(use_enable mug gtk) \
+		$(use_enable mug webkit) \
+		$(use_enable guile)
```

Comment:
You could use an array form here... but up yours, it's not required.

Reply:
Tried reading the ebuild(5) man page and using my weak Google-fu but I couldn't find the array form of use_enable :(